### DISCUSSION TITLE
Lump Doodia, Oceanopteris, and Neoblechnum?

### DISCUSSION THREAD

**Discussion by joelnitta** [pteridogroup/ppg#79] [General]
The segregate genera of _Blechnum s.l._, _Oceanopteris_ and _Neoblechnum_ render each other [non-monophyletic in FTOL 1.6.0](https://taxonium.org/?treeUrl=https%3A%2F%2Fraw.githubusercontent.com%2Ffernphy%2Fftol_vis%2Fmain%2F_targets%2Fuser%2Ftaxonium%2Fftol_tree.tree&ladderizeTree=true&treeType=nwk&metaUrl=https%3A%2F%2Fraw.githubusercontent.com%2Ffernphy%2Fftol_vis%2Fmain%2F_targets%2Fuser%2Ftaxonium%2Fftol_data.csv&metaType=meta_csv&xType=x_dist&color=%7B%22field%22%3A%22meta_family%22%7D&config=%7B%22source%22%3A%22GenBank%20release%20259%22%2C%22title%22%3A%22FTOL%20v1.6.0%22%7D&srch=%5B%7B%22key%22%3A%22aa1%22%2C%22type%22%3A%22name%22%2C%22method%22%3A%22text_match%22%2C%22text%22%3A%22Doodia_maxima%22%2C%22gene%22%3A%22S%22%2C%22position%22%3A484%2C%22new_residue%22%3A%22any%22%2C%22min_tips%22%3A0%7D%5D).

Specifically, this is caused by _Doodia maxima_ clustering with _Oceanopteris gibba_ and _Neoblechnum brasiliense_.

![Screenshot 2024-04-12 at 13-34-12 FTOL v1 6 0](https://github.com/pteridogroup/ppg/assets/13459362/8fc7465d-6110-47b8-8f1a-164ed98c36d9)

The _D. maxima_ sequence ([DMU05921](https://www.ncbi.nlm.nih.gov/nuccore/U05921)) is from Wolf et al. (1994). The specimen was cultivated at MOBOT, origin unknown.

The _O. gibba_ sequence ([MN623360](https://www.ncbi.nlm.nih.gov/nuccore/MN623360)) is a whole-plastome sequence from  Liu et al. (2020). They also used a cultivated specimen (Fairylake Bot. Garden) and as far as I can tell didn't provide any data on the original source of the material.

According to Gasper et al. (2016), _Neoblechnum_ is monotypic and sister to _Doodia_ + _Oceanopteris_, which are sister to each other. _Neoblechnum_ is only from the neotropics, whereas _Doodia_ + _Oceanopteris_ are from Malesia, Fiji, New Caledonia and Australia. However, they cite the need to study the relationships between _Doodia_ and _Oceanopteris_ in more detail, citing low branch support for the relationship of _Oceanopteris_ to _Neoblechnum_ in their tree.

I think at this point we can't say much more about this without having taxonomic experts examine the specimens in question, but I wanted to share my notes for future reference.

### References

Gasper AL, Almeida TE, Dittrich VAO, et al (2016) Molecular phylogeny of the fern family Blechnaceae (Polypodiales) with a revised genus-level treatment. Cladisticsi. https://doi.org/10.1111/cla.12173 [pdf](https://github.com/pteridogroup/ppg/files/14953980/GASPER.et.al._2016_Molecular.phylogeny.of.the.fern.family.Blechnaceae.Polypodiales.with.a.revised.genus-level.treatment.pdf)

Liu S, Wang Z, Wang H, et al (2020) Patterns and Rates of Plastid rps12 Gene Evolution Inferred in a Phylogenetic Context using Plastomic Data of Ferns. Sci Rep 10:9394. https://doi.org/10/gg23kn [pdf](https://github.com/pteridogroup/ppg/files/14953976/Liu.et.al_2020_Patterns.and.Rates.of.Plastid.rps12.Gene.Evolution.Inferred.in.a.Phylogenetic.pdf)

Wolf PG, Soltis P, Soltis D (1994) Phylogenetic relationships of dennstaedtioid ferns: evidence from rbcL sequences. Molecular Phylogenetics and Evolution 3:383–392 [pdf](https://github.com/pteridogroup/ppg/files/14953981/Wolf_et_al_1994_Phylogenetic_relationships_of_dennstaedtioid_ferns.pdf)

**Comment by SoniaMolino** [comment]
Thanks for pointing that out Joel, in fact Michael Sundue and I are working on this - stay tuned! Also, I take this opportunity to make an appeal, if anyone has access to specimens of taxa within this group I would be happy to talk about it.

**Reply by DrAshleyField** [reply]
Hi @SoniaMolino and Michael, I have access to, and in several cases already have target sequence capture data of the Australian _Doodia_ and a couple of _Oceanopteris_ if this helps. @joelnitta the Blechnaceae gurus will assist further, but it is my understanding there has been discussion over the years that _Doodia maxima_ is of hybrid origin. Ash

**Reply by SoniaMolino** [reply]
That's great, Ashley! I'll send you and email :)
